packages/filebrowser/src/listing.ts

```diff
@@ -2888,6 +3278,34 @@ export namespace DirListing {
       return dragImage;
     }
 
+    /**
+     * Factories for individual parts of the item.
+     */
+    protected _itemFactories = {
```

Comment:
```suggestion
    protected itemFactories = {
```

(convention for `protected` accessors)